server/subscribe.go

```diff
@@ -265,6 +267,86 @@ func (p *Plugin) editChannelSubscription(modifiedSubscription *ChannelSubscripti
 	})
 }
 
+func inAllowedGroup(inGroups []*jira.UserGroup, allowedGroups []string) bool {
```

Comment:
OT: we can really use a `type [String]Set ...` with a `func (s Set) Intersect(Set) Set` in >2 places now. @mickmister

Reply:
The lack of a set type in go hurts my brain.